packages/fluent-theme/src/fluent/styles/SearchBox.styles.ts

```diff
@@ -1,15 +1,65 @@
 import { ISearchBoxStyleProps, ISearchBoxStyles } from 'office-ui-fabric-react/lib/SearchBox';
 
 export const SearchBoxStyles = (props: ISearchBoxStyleProps): Partial<ISearchBoxStyles> => {
-  const { theme } = props;
+  const { theme, hasFocus, hasInput, underlined } = props;
   const { effects, palette } = theme;
 
   return {
     root: [
       {
         borderRadius: effects.roundedCorner2,
-        borderColor: palette.neutralSecondaryAlt
+        borderColor: palette.neutralTertiary,
+        paddingTop: 0,
+        paddingBottom: 0,
+        selectors: {
+          ':hover ': {
+            borderColor: palette.neutralPrimary
+          },
+          ':active': {
+            borderColor: palette.themePrimary
+          }
+        }
+      },
+      !hasFocus &&
+        hasInput && {
+          selectors: {
+            '.ms-SearchBox-iconContainer': {
+              width: 0
```

Comment:
What's the purpose of `width: 0`? Is this meant to hide the icon container?

Reply:
Exactly! However I've since changed it to 4px to mimic Fabric 6's spec (to hide on ONLY focus)